plugins/modules/web_infrastructure/jira.py

```diff
@@ -158,6 +158,7 @@
      - This is a free-form data structure that can contain arbitrary data. This is passed directly to the JIRA REST API
        (possibly after merging with other required data, as when passed to create). See examples for more information,
        and the JIRA REST API for the structure required for various fields.
+     - When passed to comment, the data structure is merged at the first level. Useful to add JIRA properties for example.
```

Comment:
```suggestion
     - When passed to comment, the data structure is merged at the first level since community.general 4.6.0. Useful to add JIRA properties for example.
```